docker/Dockerfile-17.05.0

```diff
@@ -15,4 +15,10 @@ RUN apt-get -y update && \
     apt-get -y update && \
     apt-get -y install docker-ce=17.05.0~ce-0~ubuntu-xenial
 
+ENV CRED_HELPER_VERSION=1.4.1
+
+RUN  curl -L https://github.com/GoogleCloudPlatform/docker-credential-gcr/releases/download/v${CRED_HELPER_VERSION}/docker-credential-gcr_linux_amd64-${CRED_HELPER_VERSION}.tar.gz  | tar xz -O docker-credential-gcr > /usr/local/bin/docker-credential-gcr && chmod +x /usr/local/bin/docker-credential-gcr
```

Comment:
Can we split this across lines for readability?

Reply:
Done